validator/keymanager/v2/direct/direct.go

```diff
@@ -123,6 +123,11 @@ func UnmarshalConfigFile(r io.ReadCloser) (*Config, error) {
 	return cfg, nil
 }
 
+// MarshalConfigFile returns a marshaled configuration file for a keymanager.
+func MarshalConfigFile(ctx context.Context, cfg *Config) ([]byte, error) {
```

Comment:
The reason this is now a top-level method is because it doesn't have any dependencies. Additionally, it makes it easy to marshal a keymanager config without needing to call `NewKeymanager` first.